config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml

```diff
@@ -110,7 +110,7 @@ presubmits:
       - gh-pages
       - release-0.2
     path_alias: sigs.k8s.io/cluster-api
-    always_run: true
+    run_if_changed: '^((api|bootstrap|cmd|config|controllers|controlplane|errors|exp|feature|hack|scripts|test|third_party|util)/|main\.go|go\.mod|Dockerfile|Makefile)'
```

Comment:
Is `/|main\.go` right - should there be a slash before the pipe?

Reply:
Yes, it shows they are directory. Similar to CAPV regex:

Reply:
Thanks, @fabriziopandini straightened me out 😄